Question1/occurence.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
+
+<script>
+	function checkOccurence()
+	{
+		var str = document.Form1.txt1.value;
+    		var arr = new Array();
+		
+		arr = str.split(",");
+
+		for (a in arr ) 
+		{
+	    	arr[a] = parseInt(arr[a], 10);	
+		}
+		var count=0;
+
+		for(let i=0;i<10;i++)
+		{
+			for(let j=0;j<arr.length;j++)
+			{
+				if(i==arr[j])
+					count++;
+			}
+			if(count>0)
+			{
+				document.writeln(i+" : "+count);
+				count=0;
+			}
+		}
+	}
+</script>
+	
+</head>
+<body>
+	<form name="Form1">
+		Enter comma Separated vaues:
+		<input type="text" name="txt1"><br><br>
```

Comment:
The name given to the elements are not proper. txt1 doest not gives a proper idea